packages/react-charting/src/components/TreeChart/TreeChart.test.tsx

```diff
@@ -0,0 +1,24 @@
+jest.mock('react-dom');
+import * as React from 'react';
+import * as renderer from 'react-test-renderer';
+import { IDataStructure, TreeChart } from './index';
+
+const treeData: IDataStructure = {
```

Comment:
can you create snapshot test for 3 layer chart as well

Reply:
it will be updated in new changes